core/config/locales/en.yml

```diff
@@ -1281,6 +1281,7 @@ en:
     price_sack: Price Sack
     process: Process
     product: Product
+    product_name: Product Name
```

Comment:
@MatthewKennedy I think we can avoid adding a new translation key by combining `product` + `name` translation here :)

Reply:
Yeah sure, I'll sort that.

Reply:
@damianlegawiec I'll drop that back to just `:name` if you want?

I was just thinking then as I was concatenating keys, if we build a string from separate translation keys, it might not translate well out of its context of use, if you were to use an auto translation service like translation.io on your app.

`Product number` => `Numéro de produit`
`product` + `'number`  => `Produit Numéro`

What would you prefer? I can concat the keys, drop it back to just `:name` or leave it with the new translation key?

Reply:
Let's leave it as product_name then as it's best for UX 👍